BuildingSync.xsd

```diff
@@ -3440,6 +3484,7 @@
             <xs:enumeration value="Frequency"/>
             <xs:enumeration value="Power"/>
             <xs:enumeration value="Power Factor"/>
+            <xs:enumeration value="Emissions"/>
```

Comment:
Is this the appropriate value for this enum?

Reply:
I think so, or maybe `Greenhouse Gas Emissions`